service/core/client/src/main/scala/com/lightbend/lagom/internal/client/WebSocketClient.scala

```diff
@@ -46,7 +46,7 @@ private[lagom] abstract class WebSocketClient(environment: Environment, eventLoo
 
   // Netty channel groups are used to track open channels (connections), they automatically clean themselves up when
   // the channels close, and can be used to force all channels to close.
-  val channelGroup = new CleanupChannelGroup
```

Comment:
This class was removed in the latest AsyncHttpClient, but it was never necessary to use.